mesh/settings.py

```diff
@@ -62,6 +64,7 @@
 ]
 
 CORS_ORIGIN_WHITELIST = [
+    #allow request from localhost port 3000 for testing
```

Comment:
What are these comments for?

Reply:
If I understand correctly when I launched both the backend and frontend at the same time locally, the frontend would have a different domain. Adding the CORS_ORIGIN_WHITELIST allowed the frontend to request to the backend. 

That comment was just a reminder that the backend currently allows requests from that port and I'm not too sure if this needs to be removed eventually.

This is based on a Django extension from [here](https://github.com/adamchainz/django-cors-headers) although the configuration setting is now called CORS_ALLOWED_ORIGINS

Reply:
Ah yea it just seemed redundant to me since just below it it says "localhost:3000"

Reply:
its removed. 3e2025d